pkgs/development/python-modules/cheetah3/default.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "Cheetah3";
+  version = "3.2.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "ececc9ca7c58b9a86ce71eb95594c4619949e2a058d2a1af74c7ae8222515eb1";
+  };
+
+  doCheck = false; # Circular dependency
+
+  meta = {
+    homepage = http://www.cheetahtemplate.org/;
```

Comment:
Quotes around the url https://github.com/NixOS/rfcs/pull/45

Reply:
done